oaknational/Oak-Web-Application
# feat: integrate contact page with CMS

## Description
- Wire up contact-us page to CMS
- Add ability to trigger a pre-defined JS function from a portable text annotation (similar to `internalLink`), in this case for triggering the cookie popup

## Issue(s)

Progress on #769 
Fixes #776

## How to test

Go to https://deploy-preview-787--oak-web-application.netlify.thenational.academy/contact-us
- It should look the same as the current hard-coded implementation
- Clicking "manage cookie settings" should open the consent manager

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
